browser/ui/webui/ethereum_remote_client/components/opt-in/style.ts

```diff
@@ -5,14 +5,16 @@
 
 import styled from 'styled-components'
 
-const isDarkTheme = (p: any) => {
-  return p.theme.name === 'Brave Dark'
+const darkThemeMatcher = window.matchMedia('prefers-color-scheme: dark')
+
+const isDarkTheme = () => {
```

Comment:
Just double checking that this uses the browser theme, not the OS theme? I.e. setting brave to dark mode will trigger this?

Reply:
Yes, css respects the choice made in brave settings (either OS or fixed to a specific theme)